catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/endpoint/CometdEndpoint.java

```diff
@@ -88,11 +89,11 @@
      */
     public CometdEndpoint(CometdServlet cometdServlet, CatalogFramework framework,
             FilterBuilder filterBuilder, FilterAdapter filterAdapter, PersistentStore persistentStore,
```

Comment:
It's in the blueprint: catalog/ui/search-ui/search-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml